lib/tasks/package.rake

```diff
@@ -0,0 +1,107 @@
+require 'tempfile'
+require 'English'
+
+def check_git
+  modified = `git ls-files -m --exclude-standard .`
+  if !modified.empty?
+    raise "Warning: uncommitted changes!\n\n#{modified}\n"
+  else
+    puts 'No uncommitted changes detected.'
+  end
+end
+
+def build_gem(package_dir)
+  gemfilename = "suse-connect-#{SUSE::Connect::VERSION}.gem"
+
+  `rm *.gem` if Dir['*.gem'].any?
+  `gem build suse-connect.gemspec`
+
+  raise 'Gem build failed.' unless $CHILD_STATUS.exitstatus.zero?
+
+  Dir.chdir package_dir.to_s
```

Comment:
Why change dir half-way through? You can just copy from here into `./package` ... safes a line and stays in workdir.

Reply:
This is because there might be another version of the gem in the `package` directory from the previous checkout. Maybe should have added a comment.